cmd/utils.go

```diff
@@ -85,3 +86,17 @@ func validateName(name string) error {
 	return nil
 
 }
+
+// validateStoragePath will validate storagePath, if there is any existing storage with similar path, it will give an error
+func validateStoragePath(client *occlient.Client, storagePath, componentName, applicationName string) error {
+	storeList, err := storage.List(client, componentName, applicationName)
+	if err != nil {
+		log.Debugf("Unable to list storage")
```

Comment:
Shouldn't the error be returned and further execution be stopped?

Reply:
if there is no existing storage, it will throw an error, so I didn't returned error here

Reply:
But I think since the function returns the error, it's better to throw the error for later use in the calling function

Reply:
Let's say there is no existing storage, it is gonna definitely throw an error. but that's perfectly fine situation. we are just validating here.

Reply:
But storage.List might return error (e.g due to PVC error), validateStoragePath won't return and stop execution and it will return nil. So checkError() won't catch it and the program will still try to create the PVC.
```
[mdas@localhost odo]$ ./odo storage create backend --path /data --size 100M
Added storage backend to wildfly
[mdas@localhost odo]$ ./odo storage create backend-1 --path /data --size 100M
DeploymentConfig "wildfly-app" is invalid: spec.template.spec.containers[0].volumeMounts[2].mountPath: Invalid value: "/data": must be unique
```
Here I deleted the PVC to replicate the storage.List() PVC error, the error was not caught and the program still tried to make the PVC.

Reply:
@mik-dass  Sorry my bad , I was little confused. updating the PR accordingly

Reply:
No problem, my bad too, I should have explained better at the start. Anyways rest looks good to me :+1: